diff --git a/README.md b/README.md
index 110632b..3cccbff 100644
--- a/README.md
+++ b/README.md
@@ -1,5 +1,3 @@
-#v1.14.26
+#v1.14.27
 
-- Bug ao trocar de atividades confusão de regras ajustado.
-- Inclusão de job extra: efetuar novo teste, aqui está funcionando certinho.
-- Score do cliente: não há mais espaço na linha, se apertar alguma informação vai ocasionar mais conflitos. O que sugere? 
+- Sugestão para essa sinalização: https://material.angular.io/components/chips/overview
diff --git a/src/app/schedule/schedule-form/schedule-form.component.ts b/src/app/schedule/schedule-form/schedule-form.component.ts
index d6b3ec6..b32ef09 100644
--- a/src/app/schedule/schedule-form/schedule-form.component.ts
+++ b/src/app/schedule/schedule-form/schedule-form.component.ts
@@ -148,7 +148,7 @@ export class ScheduleFormComponent implements OnInit {
         }
       }))
 
-    this.subscriptions.push(this.scheduleForm.controls.duration.valueChanges
+    this.scheduleForm.controls.duration.valueChanges
       .pipe(debounceTime(500))
       .subscribe(value => {
         if (value >= 5) {
@@ -158,7 +158,7 @@ export class ScheduleFormComponent implements OnInit {
         }
         this.addValidationBudget()
         this.calculateNextDate()
-      }))
+      })
 
     this.addValidationBudget()
   }
